sdk/samples/iot/README.md

```diff
@@ -129,21 +133,27 @@ To run the samples, ensure you have the following programs and tools installed o
     </p>
     </details>
 
-- Have the latest version of [CMake](https://cmake.org/download) installed. On Linux, this installation may also take an extended amount of time (~20-30 minutes).
+- Have CMake installed.
```

Comment:
Maybe:
```suggestion
- Have the minimum required version of CMake installed.
```

https://github.com/Azure/azure-sdk-for-c/blob/8eb5a1f124ab6d811bc076480691896afbaf6f2d/CMakeLists.txt#L4